package.json

```diff
@@ -33,9 +33,13 @@
     "js-lint": "eslint js/ && eslint --config js/tests/.eslintrc.json --env node build/ Gruntfile.js",
     "js-lint-docs": "eslint --config js/tests/.eslintrc.json assets/js/ sw.js",
     "js-compile": "npm-run-all --parallel js-compile-*",
-    "js-compile-bundle": "shx cat js/src/util.js js/src/alert.js js/src/button.js js/src/carousel.js js/src/collapse.js js/src/dropdown.js js/src/modal.js js/src/scrollspy.js js/src/tab.js js/src/tooltip.js js/src/popover.js | shx sed \"s/^(import|export).*//\" | babel --filename js/src/bootstrap.js | node build/stamp.js > dist/js/bootstrap.js",
-    "js-compile-plugins": "babel js/src/ --out-dir js/dist/ --source-maps",
-    "js-minify": "uglifyjs --config-file build/uglifyjs.config.json --output dist/js/bootstrap.min.js dist/js/bootstrap.js",
+    "js-compile-standalone": "rollup --environment BUNDLE:false --config build/rollup.config.js",
+    "js-compile-bundle": "rollup --environment BUNDLE:true --config build/rollup.config.js",
+    "js-compile-plugins": "babel --no-babelrc js/src/ --out-dir js/dist/ --source-maps --presets=es2015 --plugins=transform-es2015-modules-strip",
+    "postjs-compile": "babel dist/js/bootstrap.js -o dist/js/bootstrap.js && babel dist/js/bootstrap.bundle.js -o dist/js/bootstrap.bundle.js && node build/stamp.js",
```

Comment:
Is this supposed to be included in another task?

Reply:
OK I think I get it now. This task is supposed to be the main JS build task. I think we should keep `js-compile` as the default one with stamp. I find the current tasks a little confusing :/

Reply:
that's cannot be possible because `js-compile` run all the `js-compile*` tasks so I should wait the end of those tasks to stamp our dist files